Scripts/developer_scripts/create_new_release

```diff
@@ -290,9 +302,15 @@ fi
 # Build public release version
 if [ -n "$DO_PUBLIC" ]; then
     echo "Making the public version of the tarball"
-    public_release_number="1${major_nr}${minor_nr}${bugfix_nr}1000"
+    if [ -n "$BETA" ]; then
+      public_release_number="1${major_nr}${minor_nr}${bugfix_nr}09${BETA}0"
+    else
+      public_release_number="1${major_nr}${minor_nr}${bugfix_nr}1000"
+    fi
     public_release_version="${MAJOR_NUMBER}.${MINOR_NUMBER}${BUGFIX_STRING}"
-    if [ -r "${NUMBERS_DIR}/public_release_name" ]; then
+    if [ -n "$BETA" ]; then
+	public_release_name="CGAL-${public_release_version}-beta${$BETA}"
```

Comment:
Well... it was not really tested. I tried yesterday the following command:
```shellsession
$ cd $HOME/CGAL/create_internal_release && /usr/bin/time $HOME/bin/create_release $HOME/CGAL/branches/integration.git $HOME/CGAL/branches/empty-dir --public --beta 2 --do-it
```
and it failed with:
```
/home/lrineau/CGAL/last-release-branch/Scripts/developer_scripts/create_new_release: line 312: CGAL-${public_release_version}-beta${$BETA}: bad substitution
```
and indeed that `${$BETA}` is not a valid syntax.